### DISCUSSION TITLE
ITwin V3 IModel user unauthorized [evercam]

### DISCUSSION THREAD

**Discussion by salah-walid** [iTwin/itwinjs-core#2965] [Q&A] [answered]
Hi,

Here at evercam we are trying to test ITwin V3, but it's saying that the user is unauthorized even though the user is added to the IModel in IModelHub

![image](https://user-images.githubusercontent.com/29980658/147829593-85c6b556-70ff-412e-8f63-b9b3877e0a40.png)

Are we missing something ?

**Comment by aruniverse** [comment, marked as answer]
Have you added the new scope required? `imodels:read` Take a look at, the [breaking changes on the viewer side](https://github.com/iTwin/viewer/blob/next/NextVersion.md#new-scopes-required)

**Reply by oussamabonnor1** [reply]
Thank you! It's worth noting that adding the ITwin Management API was needed.

**Reply by salah-walid** [reply]
Thank you, we managed to make it work
